core/commands/ls.go

```diff
@@ -57,6 +65,8 @@ The JSON output contains type information.
 	Options: []cmdkit.Option{
 		cmdkit.BoolOption("headers", "v", "Print table headers (Hash, Size, Name)."),
 		cmdkit.BoolOption("resolve-type", "Resolve linked objects to find out their types.").WithDefault(true),
```

Comment:
Since this is now deprecated, we should probably say so (and say to use `--type` instead)!